src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts

```diff
@@ -92,7 +92,7 @@ export class JJCountComponent implements OnInit {
 
     // initialize if no value
     if (this.jjDisputedCount) {
-      if (this.jjDisputedCount.totalFineAmount) {
+      if (this.jjDisputedCount.totalFineAmount != null) {
```

Comment:
Why add not equal null and what if it is zero

Reply:
zero is a valid value, dont want other branch if zero.  next line will  not cause errors.  dividing by zero causes errors but not dividing zero by a number.